client/tests/test_client_page.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from client.models import Client
+from django.contrib.auth.models import User
+from client.views import client_check
+from unittest.mock import Mock
+
+
+@pytest.fixture
+def non_client_mock():
+    mock = Mock()
+    mock.client_account = User.objects.create_user(username='Asaf102',
+                                                   password='litza')
+    return mock
+
+
+class TestClientLogin:
+    @pytest.mark.django_db()
+    def test_success_entry(self, client, saved_client0):
+        client.force_login(user=saved_client0.client_account)
+        assert saved_client0 in Client.objects.all()
+        respone = client.get('/client/')
+        assert respone.status_code == 200
+
+    @pytest.mark.django_db()
+    def test_fail_entry(self, client, non_client_mock):
+        client.force_login(user=non_client_mock.client_account)
+        assert non_client_mock not in Client.objects.all()
+        respone = client.get('/client/')
+        assert respone.status_code != 200
+
+    @pytest.mark.django_db()
+    def test_client_check_false(self, non_client_mock):
+        assert non_client_mock not in Client.objects.all()
+        assert not client_check(non_client_mock.client_account)
```

Comment:
`assert not client_check(...)` is equivalent to `assert bool(client_check(...)) is False`. You want to assert the value of the actual output rather than "booling" that value, as there are multiple things which bool to `False`.
```suggestion
        assert client_check(non_client_mock.client_account) is False
```